loopbackStarter/manifest.yml

```diff
@@ -0,0 +1,10 @@
+applications:
+- path: .
+  memory: 256M
+  instances: 1
+  domain: mybluemix.net
+  name: loopback-starter-app
+  host: loopback-starter-app
+  disk_quota: 256M
```

Comment:
and actually, you don't need to put `applications:` because there is only one application. the entire manifest.yml file can look like this:
```
path: .
memory: 256M
instances: 1
domain: mybluemix.net
name: loopback-starter-app
host: loopback-starter-app
disk_quota: 256M
services:
- loopback-mongodb-starter
env:
  NODE_ENV: bluemix
```